pde2e-image/common/windows/common.ps1

```
@@ -1,3 +1,400 @@
+        Write-Output "No standard output..."
+    }
+
+    if (Test-Path $errorFile) {
+        Write-Output "Standard Error: $(Get-Content -Path $errorFile)"
+    } else {
+        Write-Output "No standard error..."
+    }
+}
+
+# Check if something in on PATH
+function Is-In-Path($directory, $scope) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Is-In-Path' uses an unapproved verb.

(PSUseApprovedVerbs)